README.md

```diff
@@ -57,8 +46,257 @@ Express has extensive support for template rendering and can load and leverage H
+  5. In src/views create a ```layouts``` directory.
+  6. In src/views create a ```partials``` directory.
+
+
+### Loading up main page
+
+1. Create a ```home.hbs``` in the views directory
+1. Navigate to public/index.html. Copy all the contents to ```home.hbs```.
+1. Delete public/index.html.
+1. In src/views/layouts create a main.hbs file and insert the following code:
+    ```hbs
+    {{{ body }}} 
+    ```
+    Body will be the hbs file we are rendering. So when we go to the '/' route it will be ```home.hbs```.
+    6. To start the server in development mode use the npm script from the package.json by typing this in your terminal:
+    ```bash
+    npm run start:watch
+    ```
+
+    Load up the server and go to localhost:3000 in your browser; expect to see a 404 error. We are now going to serve up the ```home.hbs``` file.
+
+7. Create a ```home.js``` in src/controllers
+
+8. In ```home.js``` add the following code:
+    ```js
+    exports.get = (req, res) => {
+      res.render('home');
+    };
+    ```
+    Here ```res.render``` renders a view and sends a rendered HTML string to the client.
+
+9. In src/controllers/index.js require in ```home.js```.
+
+10. Add the following line after files are required in:
+    ``` js
+    router.get('/', home.get);
+    ```
+
+## Split up the contents into partials
+We are now going to create partials to simplify our website. Partials allow us to reuse snippets of HTML preventing duplication.
+1. In src/views/partials create the following files:
+    * htmlHead.hbs
+    * header.hbs
+    * footer.hbs
+2. From home.hbs cut:
+    * head to htmlHead.hbs
+    * header to header.hbs
+    * footer to footer.hbs
+3. The next stage is to continue creating the ```main.hbs``` layout and inserting in the partials we have created. Copy across everything inside ```home.hbs``` except for the section tags and their content.
+4. Place ```{{{body}}}``` inside the ```<main>``` tags.
+5. Now you can insert the partials you have created into the ```main.hbs``` file. Use the [handlebars documentation](http://handlebarsjs.com/) to find out how.
+
+## Creating The Fruits Page
+1. In src/views create a ```fruits.hbs``` file
+1. Open public/fruits.html and copy its contents to the ```fruits.hbs``` file
+1. Delete ```fruits.html```
+1. Delete everything from ```fruits.hbs``` that you have created partials for.
+
+**See all the many list items? We will create these programmatically using handlebars.**
+
+5. Delete all but one of the list items.
+
+**Open up src/model/index.js and take a look. We will use this list for creating out fruits page.**
+
+6. Now go back to src/controllers/fruits.js and require in the model of fruits.
+    ```js
+    const fruits = require('./../model/index');
+    ```
+
+    ```res.render``` also takes an object as an argument which can be used in the hbs file to insert content. Content will be available as a variable named as the object key(s).
```

Comment:
I am not convinced of the wording: `insert content`. More we are parameterising (dont use that word either) the view.

Maybe more like: `customise the view`

Reply:
ie something more like 
  > ```res.render``` also takes an object as a optional second argument. This can be used to customise the view. Within the view values passed in on the options object will be available as a variables named after the object keys.

Reply:
Amended https://github.com/foundersandcoders/express-handlebars-workshop/pull/11/commits/6efbeb64f2dc03cde2bb1fc5b27f13a6f7be54d2